docs/plans/2026-05-18-ROB-264-finnhub-us-earnings-range-ingest.md

```
@@ -0,0 +1,1526 @@
+
+# Narrow recovery of a failed tail
+uv run python -m scripts.ingest_market_events \
+  --source finnhub --category earnings --market us \
+  --from-date 2026-07-10 --to-date 2026-07-17
+```
+
+On a 429, the CLI exits with code `2` and prints a summary line whose JSON
+includes `"error": "finnhub_quota_exceeded"`. No partition rows are mutated
+on the failed call; partition state from previous successful runs is
+preserved as-is.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)